src/stories/Library/content-list-item/content-list-item.scss

```diff
@@ -67,23 +67,27 @@ $_stacked-reduce-width: 20px;
   }
 }
 
-.event-list-item__image-container {
+.content-list-item__image-container {
   @extend %identity-placeholder-pseudo;
   grid-area: image;
   position: relative;
   width: 100%;
   aspect-ratio: 4 / 3;
-}
 
-.event-list-item__image {
-  width: 100%;
-  height: 100%;
-  object-fit: cover;
-  position: relative;
-  z-index: $z-10;
+  // We want to treat everything inside the image-container as image related.
+  // To make object-fit work, it is important that everything has 100%
+  // height and width set. We cannot know 100% what markup Drupal will use,
+  // so we'll use a wildcard here.
+  * {
```

Comment:
It is hard for me to know what the exact problem is here.

The intention is for the design system to dictate the markup and for the consuming system to follow. I do not know what situation forces us to leave it as this.  Perhaps it would be easier to look at it on a screen together.